platform-tests/example-apps/healthcheck/db.go

```diff
@@ -35,7 +44,7 @@ func testDBConnection(ssl bool) error {
 		dbURL.RawQuery = dbURL.RawQuery + "&sslmode=disable"
 	}
 
-	db, err := sql.Open("postgres", dbURL.String())
+	db, err := sql.Open(service, dbURL.String())
```

Comment:
Aaagh! Not sure, why it has imported the mysql driver in platform-tests/src/acceptance/rds_broker_test.go - which you already commented on.

Sorry about that. Fixing.